src/FullscreenQuad.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "Drawable.h"
+#include <VertexBufferDX11.h>
+#include <IndexBufferDX11.h>
+
+class FullscreenQuad
```

Comment:
Commit description of why we will need this class / what it will be used for, would be nice